src/components/loginForm/index.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Button from "../../commons/buttons"
+const Login = (props) => {
+  const { onSubmit, errors, onRegister, onPassword } = props;
+  return (
+    <div className="login-wrapper">
+      <div className="col-md-12">
+        <h3>Author's Haven</h3>
```

Comment:
Thank you @wasibani-roy for the good work, but please put `Author's` in double quotes in like `{"Author's"}`. In order to remove that error.

Reply:
I have made the change